permissions/plugin-extra-tool-installers.yml

```diff
@@ -2,11 +2,13 @@
 name: "extra-tool-installers"
 github: "jenkinsci/extra-tool-installers-plugin"
 issues:
-  - jira: '17920'  # extra-tool-installers-plugin
```

Comment:
Probably worth filing a helpdesk issue to archive the component, otherwise people may submit issues there.

Reply:
Sure I will